storage/storage_test.go

```diff
@@ -68,23 +68,23 @@ func newStore(t *testing.T) storage.Store {
 	}
 	run := filepath.Join(wd, "run")
 	root := filepath.Join(wd, "root")
-	uidmap := []idtools.IDMap{{
+	Transport.SetDefaultUIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getuid(),
 		Size:        1,
-	}}
-	gidmap := []idtools.IDMap{{
+	}})
+	Transport.SetDefaultGIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getgid(),
 		Size:        1,
-	}}
+	}})
 	store, err := storage.GetStore(storage.StoreOptions{
 		RunRoot:            run,
 		GraphRoot:          root,
 		GraphDriverName:    "vfs",
 		GraphDriverOptions: []string{},
```

Comment:
I think the only risk there is for consumers of both libraries that don't use the version of `storage` that we specify in the `vendor.conf` in this package.  The CI system should help catch that sort of change when we update the version listed there, so I think it's worth adding that.